### ISSUE TITLE
PHP and Javascript library does not match

### ISSUE THREAD

**Issue by arunvar008** [dropbox/zxcvbn#283]
Hi, I am facing an issue with js and php library.

I passed password "arunvarghese@1985"

in js, it got score : 4 ..but in php it s 3.. can someone help.. it s not identical

I am using https://github.com/dropbox/zxcvbn for js
and https://github.com/bjeavons/zxcvbn-php for php

**Comment by mkopinsky** [comment]
Please open an issue in the PHP repo.

**Comment by bjeavons** [comment]
@arunvar008 Yes, please file an issue at https://github.com/bjeavons/zxcvbn-php/issues and include information on your environment, including details on how you installed Zxcvbn-PHP (eg. from composer or otherwise).

Environment information will be helpful because I am not able to reproduce a score difference using HEAD of Zxcvbn-PHP (v1.0.0) and the hosted JS version at https://lowe.github.io/tryzxcvbn/. Both score a 4 on that password.
